components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/WEB-INF/web.xml

```diff
@@ -327,6 +327,55 @@
         <location>/generic-exception-response.jsp</location>
     </error-page>
 
+    <!-- custom error pages -->
+    <error-page>
+        <error-code>400</error-code>
+        <location>/errors/error_400.html</location>
+    </error-page>
+    <error-page>
+        <error-code>401</error-code>
+        <location>/errors/error_401.html</location>
+    </error-page>
+    <error-page>
+        <error-code>403</error-code>
+        <location>/errors/error_403.html</location>
+    </error-page>
+    <error-page>
+        <error-code>404</error-code>
+        <location>/errors/error_404.html</location>
+    </error-page>
+    <error-page>
+        <error-code>405</error-code>
+        <location>/errors/error_405.html</location>
+    </error-page>
+    <error-page>
+        <error-code>408</error-code>
+        <location>/errors/error_408.html</location>
+    </error-page>
+    <error-page>
+        <error-code>410</error-code>
+        <location>/errors/error_410.html</location>
+    </error-page>
+    <error-page>
+        <error-code>500</error-code>
+        <location>/errors/error_500.html</location>
+    </error-page>
+    <error-page>
+        <error-code>502</error-code>
+        <location>/errors/error_502.html</location>
+    </error-page>
+    <error-page>
+        <error-code>503</error-code>
+        <location>/errors/error_503.html</location>
+    </error-page>
+    <error-page>
+        <error-code>504</error-code>
+        <location>/errors/error_504.html</location>
+    </error-page>
+    <error-page>
+        <location>/errors/error.html</location>
+    </error-page>
```

Comment:
Check if this overrides the error page which is configured for exception.

Reply:
No I checked throwing an exception, It gives the generic-exception-response.jsp, not the error.html